client/src/pages/UserLanding/UserLanding.jsx

```diff
@@ -58,20 +61,43 @@ const UserLanding = () => {
     const searchResults = searchResponse;
 
     setNextPageToken(searchResults.nextPageToken);
+
+    // 50 items
     setSearchedVideos(searchResults.items);
   };
 
   const loadMoreSearch = async () => {
-    const searchResults = await getSearchVideos(userSearchTerm, nextPageToken);
-    setNextPageToken(searchResults.nextPageToken);
-    setSearchedVideos((prevVideos) => [...prevVideos, ...searchResults.items]);
+    setSearchVideosToShow((prevValue) => prevValue + 2);
   };
 
+  if (searchVideosToShow % 50 === 0) {
+    const getMoreSearchVideos = async () => {
+      const searchResults = await getSearchVideos(
+        userSearchTerm,
+        nextPageToken
+      );
+      setSearchVideosToShow((prevValue) => prevValue + 2);
+      setNextPageToken(searchResults.nextPageToken);
+      setSearchedVideos((prevVideos) => [
+        ...prevVideos,
+        ...searchResults.items,
+      ]);
+    };
+    getMoreSearchVideos();
+  }
+
   const loadMoreWorkouts = () => {
     setWorkoutsToShow((prevValue) => prevValue + 2);
   };
 
-  console.log(workoutsToShow);
+  // MANAGE WHEN TARGET AREA FILTER  IS UPDATED
+  const handleTargetAreaFilterChange = (e) => {
+    setTargetAreaFilter(e.target.value);
+  };
+
+  useEffect(() => {
```

Comment:
I would actually recommend combining this - 
get rid of the useEffect, and just call `getSavedWorkouts(e.target.value)` w/in the handleTargetAreaFilterChange b/c you know that you're going to have to update the saved workouts when it's called. This will be more efficient & reduce the number of re-renders needed.

Can explain why in 1/1 if curious but recently React has given updated `useEffect` guidance - recommend reading [this](https://react.dev/learn/you-might-not-need-an-effect) if you are curious!

Reply:
I implemented this change and removed the targetAreaFilter piece of state. This input is no longer a controlled component and I would like to talk about when a component should be controlled vs when it shouldn't be